### PR TITLE
docs(guides): update npx introduction

### PR DESCRIPTION
**Pull request by chenxsan**
I think `npx` is introduced in [npm 5.2.0](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).

### CODE REVIEW

**Comment by chenxsan** [comment]
I just double checked the changelog from [npm](https://github.com/npm/npm/blob/latest/CHANGELOG.md#v520-2017-07-05):

> v5.2.0 (2017-07-05):

> It's only been a couple of days but we've got some bug fixes we wanted to get out to you all. We also believe that npx is ready to be bundled with npm, which we're really excited about!
npx!!!

It seems to be 5.2.0

**Comment by montogeek** [comment]
@chenxsan Correct, don't remove Node version, keep both.

**Comment by chenxsan** [comment]
@montogeek Updated.

File: src/content/guides/getting-started.md [hunk 1]
```diff
@@ -174,7 +175,7 @@ __dist/index.html__
 
 In this setup, `index.js` explicitly requires `lodash` to be present, and binds it as `_` (no global scope pollution). By stating what dependencies a module needs, webpack can use this information to build a dependency graph. It then uses the graph to generate an optimized bundle where scripts will be executed in the correct order.
 
-With that said, let's run `npx webpack` with our script as the [entry point](/concepts/entry-points) and `main.js` as the [output](/concepts/output). The `npx` command, which ships with Node 8.2 or higher, runs the webpack binary (`./node_modules/.bin/webpack`) of the webpack package we installed in the beginning:
+With that said, let's run `npx webpack` with our script as the [entry point](/concepts/entry-points) and `main.js` as the [output](/concepts/output). The `npx` command, which ships with Node 8.2/npm 5.2.0 or higher, runs the webpack binary (`./node_modules/.bin/webpack`) of the webpack package we installed in the beginning:
```
Comment (EugeneHlushko) [inline src/content/guides/getting-started.md]: suggest rephrasing into `... ships with npm 5.2.0 (Node 8.2) or higher`

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**